### DISCUSSION TITLE
Multicolumn height-alignment

### DISCUSSION THREAD

**Discussion by oxSxo** [efemkay/obsidian-modular-css-layout#63] [Q&A] [answered]
Hey,

i have a short question about alignment, when using Multicolumn callouts. When pasting the given example, my callouts vary vertical size. I would like to get them similar to the example in the matching vertical size for all columns. Im fairly new to Obsidian, what did i miss?

![image](https://github.com/efemkay/obsidian-modular-css-layout/assets/150076204/9b9488a6-050d-4ab7-988c-f207acf9d2d5)

Thank you!

**Comment by efemkay** [comment, marked as answer]
https://github.com/efemkay/obsidian-modular-css-layout/releases/tag/0.9.8

I had a minor unintended regression in 0.9.7, pls update to 0.9.8 (release link as per above).

That should fix your issue. Let me know if it doesn't

**Reply by oxSxo** [reply]
👍 Thank you! Fixed the issue!
